hw3/src/pages/home.jsx

```diff
@@ -0,0 +1,30 @@
+import Navbar from "../components/navbar";
+
+const Header = () => {
+  return (
+    <div className="text-center mt-10">
+      <p className="text-5xl font-bold font-sans text-blue-600">
+        The Countries of South America
+      </p>
+      <br />
+      <p className="mt-3 max-w-lg text-xl ml-auto mr-auto">
+        Welcome to Homework 3 website. On this website you will learn more about
+        South American countries. Navigate on our website to see list of
+        contries in South America, their population, flags and more!
+      </p>
+    </div>
+  );
+};
+
+const Home = () => {
+  return (
+    <div className="flex h-screen">
```

Comment:
The flex container might not handle the Navbar correctly if the Navbar lacks proper flexbox styling, causing layout issues. Ensuring Navbar has appropriate flex styles will resolve this.